tests/dataloaders/test_datasplitter.py

```diff
@@ -40,6 +40,79 @@ def test_datasplitter_shuffle():
     )
 
 
+def test_datasplitter_external():
+    adata = scvi.data.synthetic_iid()
```

Comment:
We should test here also all modules with specific exceptions. I.e. pyro, contrastive, scanvi, scvi. Those tests should go in the respective module tests. The error with n_unlabelled above should throw an error in the scANVI test but not in the tests here.

Reply:
Yes you are correct.
I will add those and it will be a good option to see how to run a different module.